charts/slopeCharts/LabelledSlopes.tsx

```diff
@@ -362,14 +361,12 @@ export class LabelledSlopes extends React.Component<LabelledSlopesProps> {
     }
 
     @computed get yDomain(): [number, number] {
+        const domain = this.props.yAxisOptions.domain
+        const domainDefault = this.yDomainDefault
         return [
-            this.props.yDomain[0] === undefined
-                ? this.yDomainDefault[0]
-                : this.props.yDomain[0],
-            this.props.yDomain[1] === undefined
-                ? this.yDomainDefault[1]
-                : this.props.yDomain[1]
-        ] as [number, number]
+            Math.min(domain[0], domainDefault[0]),
```

Comment:
Hm. Given that the old code checked explicitly for `undefined`, and `Math.min(1, undefined)` is `NaN`, it might be worth doing that still just to be safe.

Reply:
Interesting, didn't expect Math.min(1, undefined) to return NaN. The new axis code now returns [-Infinity, Infinity] if min or max are undefined so should be safe.